### PR TITLE
FISH-8497 add dependabot conditions

### PR DESCRIPTION
**Pull request by abdulrahim458**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -12,3 +7,30 @@ updates:
...
+      - dependency-name: "jakarta.*"
+        update-types: ["version-update:semver-minor"]
+      - dependency-name: org.eclipse.microprofile*"
+        update-types: ["version-update:semver-major"]
+      - dependency-name: "org.eclipse.microprofile*"
+        update-types: ["version-update:semver-minor"]
+  - package-ecosystem: "gitsubmodule"
+    directory: "/"
+    schedule:
+      interval: "daily"
```
Comment (Pandrex247) [inline .github/dependabot.yml:33]: Do we need to define a schedule? It seems a bit odd to scan daily to ignore everything

File: .github/dependabot.yml [hunk 2]
```diff
@@ -12,3 +7,30 @@ updates:
...
+      jakarta:
+        patterns:
+        - "jakarta.*"
+        update-types:
+        - "patch"
+      microprofile:
+        patterns:
+        - "org.eclipse.microprofile*"
+        update-types:
+        - "patch"
```
Comment (Pandrex247) [inline .github/dependabot.yml:20]: I don't think we want these grouped. I'd continue letting them being updated one at a time
```suggestion
```

File: .github/dependabot.yml [hunk 3]
```diff
@@ -12,3 +7,30 @@ updates:
...
+    ignore:
+      - dependency-name: "jakarta.*"
+        update-types: ["version-update:semver-major"]
+      - dependency-name: "jakarta.*"
+        update-types: ["version-update:semver-minor"]
+      - dependency-name: org.eclipse.microprofile*"
+        update-types: ["version-update:semver-major"]
+      - dependency-name: "org.eclipse.microprofile*"
+        update-types: ["version-update:semver-minor"]
```
Comment (Pandrex247) [inline .github/dependabot.yml:29]: Do we need separate entries for ignoring major and minor?
It's an array isn't it?
Can we not have:
```
    ignore:
      - dependency-name: "jakarta.*"
        update-types: ["version-update:semver-major", "version-update:semver-minor"]
      - dependency-name: org.eclipse.microprofile*"
        update-types: ["version-update:semver-major", "version-update:semver-major"]
```

**Review by Pandrex247 — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 3 above)
Comment (abdulrahim458) [reply .github/dependabot.yml:29]: updated in single entries

File: .github/dependabot.yml (hunk 2 above)
Comment (abdulrahim458) [reply .github/dependabot.yml:20]: removed groupng

Comment (Pandrex247) [reply .github/dependabot.yml:20]: I don't think you have?

File: .github/dependabot.yml (hunk 3 above)
Comment (abdulrahim458) [reply .github/dependabot.yml:29]: I created a new PR. Iam not able to push to this branch as its not from my Forked repo.

**Pull request closed without merging**